projects/laji/src/app/+project-form/form/named-place/np-info/np-info.component.ts

```diff
@@ -99,7 +106,7 @@ export class NpInfoComponent implements OnInit, OnChanges, AfterViewInit {
       }
 
       let pipe;
-      if (field === 'taxonIDs') {
+      if (field === 'taxonIDs' || (fieldSchema?.type === 'string' && fieldSchema.oneOf)) {
```

Comment:
This conditition `fieldSchema?.type === 'string' && fieldSchema.oneOf` was also used before. It could be moved to some service as a function `isStringEnumField(schema): boolean`

Reply:
There are only a few checks for the string enum schema type in the repo, and I didn't come up with a good place where to put the function. Creating a whole new schema utils for this simple check seems excessive. I binded the check into variables called `isEnumSchema`, should help readibility a bit.

Reply:
Ok, it sounded like a common check but maybe it's ok like that for now